contracts/modules/capital/Pool.sol

```diff
@@ -37,6 +36,9 @@ contract Pool is IPool, MasterAwareV2, ReentrancyGuard {
   uint public constant MCR_RATIO_DECIMALS = 4;
   uint internal constant MAX_SLIPPAGE_DENOMINATOR = 10000;
 
+  mapping(address => uint256) public assetsInSwapOperator;
+  uint32 internal assetsInSwapOperatorBitmap;
```

Comment:
these are not constants, move them up. the `swapOperator` uses 160 bits, could fit the 32 bits in the same slot. also `s/uint256/uint` please :innocent: